src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
         <Route path='/home' element={<Home />} />
         <Route path='/about' element={<About />} />
         <Route path='/' element={<Home/>} />
+        <Route path='/whattodo' element={<Whattodo/>} />
```

Comment:
`/` usually at the end for readabliity, move that to the end, also let's use `_` with multiple words in the url `what_to_do`